scss/main.scss

```diff
@@ -202,6 +202,10 @@ h1,
     .s2 { color: #8ae234; }
 }
 
+.copyable {
+    img { cursor: pointer; }
```

Comment:
let's put this on its own line like the others -- scss-lint has an exclusion of this, but it's only intended for the pygments styles which were copy pasted

Reply:
this can also be written without nesting as just `.copyable img {`

Reply:
done